build.gradle

```diff
@@ -23,9 +17,14 @@ repositories {
   mavenCentral()
 }
 
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+def junitJupiterVersion = '5.7.0'
```

Comment:
That's because only jupiterVersion needs to be aligned in multiple artifacts. Other dependencies use single artifact only, therefore they don't need an extra variable with the version.